### ISSUE TITLE
Add db file job_noenter_map.txt with format: JobID,FlagZone,GroupLevelBypass.

### ISSUE THREAD

**Issue by Iniciando** [HerculesWS/Hercules#1604]
https://github.com/rathena/rathena/pull/1526/commits/91799eb2b1e9187d083eda4360cf2b95991cab97

**Comment by 4144** [comment]
We not adding csv config files.
Probably you want config in libconfig format?
And for this separate files is useless too.

If really need add it, better add fields for jobs in existing config file.

**Comment by Iniciando** [comment]
No. This is a restriction that came with the creation of WOE TE, this file blocks the entry of the 3rd classes in the castle. Reference #1573

**Comment by MishimaHaruna** [comment]
If this is associated to map zones, then it should be added to the map_zone_db.

**Comment by Iniciando** [comment]
Sorry, Haru. But map_zone_db does not have the same function ... it's similar, but quite different. To the best of my knowledge, there is no file in the emulator that has such a function. So the reason for the request.

**Comment by Emistry** [comment]
same opinion with @MishimaHaruna , it should be a new field in the map_zone_db.
something look like this.
http://herc.ws/board/topic/10300-map-zone-expansion/

**Comment by Iniciando** [comment]
Now I understand what both said. You are thinking of adding a new constraint to map_zone_db. I had understood that map_zone_db already had this function, when in fact it does not yet exist. Excuse me, gentlemen.
